src/components/LabelMenus/styles.module.css

```diff
@@ -1,81 +1,75 @@
 .labelMenu {
-    width: 20%;
+  width: 20%;
 }
 
 .labelMenus {
-    height: 13rem;
-    border: solid 0.0625rem #707070;
-    background-color: #f4f4f4;
-    margin: 2.1875rem 0rem;
-    padding: 1.875rem 0rem 0rem 0rem;
+  height: 13rem;
+  border: solid 0.0625rem #707070;
+  background-color: #f4f4f4;
+  margin: 2.1875rem 0rem;
+  padding: 1.875rem 0rem 0rem 0rem;
 }
 .highlightLabel {
-    background-image: url("/highlightbar.png");
-    background-size: 0.5rem 35rem;
-    background-repeat: no-repeat;
+  background-image: url("/highlightbar.png");
+  background-size: 0.5rem 35rem;
+  background-repeat: no-repeat;
 }
 
 .labelMenuDescription {
-    flex: 1;
-    display: flex;
-    width: 5.125rem;
-    height: 0.875rem;
-    font-family: Helvetica;
-    font-size: 0.875rem;
-    font-weight: normal;
-    font-stretch: normal;
-    font-style: normal;
-    line-height: 1;
-    letter-spacing: normal;
-    text-align: left;
-    margin: 0.25rem 0rem 0rem 1.46875rem;
+  flex: 1;
+  display: flex;
+  width: 5.125rem;
+  height: 0.875rem;
+  font-family: Helvetica;
```

Comment:
Per the conversation on Thursday, let's remove all font declarations. We'll make one at the top level in `layout.css` once we decide which font, and then we'll have all pages inherit that font-family.